src/components/headers/light.js

```diff
@@ -77,7 +77,6 @@ export default ({
         <NavLink href="/">Home</NavLink>
         <NavLink href="/about">About</NavLink>
         <NavLink href="/projects">Projects</NavLink>
-        <NavLink href="/events">Events</NavLink>
```

Comment:
This should close #55. Can you make sure to close that issue when this branch gets merged?